tasks/RedHat/configuration.yml

```diff
@@ -5,18 +5,18 @@
 #    verbosity: "{{ debuglevel }}"
 #
 - name: list of required SAP Notes
-  debug: 
+  debug:
     var: __sap_hana_preconfigure_sapnotes | difference([''])
 
 # We want to disable firewalld, so we need to check if firewalld is installed
 - name: Gather facts about installed packages
   package_facts:
     manager: auto
 
-- name: include configuration actions for required sapnotes 
+- name: include configuration actions for required sapnotes
   include_tasks: "sapnote/{{ item }}.yml"
   with_items: "{{ __sap_hana_preconfigure_sapnotes | difference(['']) }}"
-     
-- include: "{{ './' + ansible_distribution + '_' + ansible_distribution_major_version + '/recommendations.yml' }}"
+
+- include_tasks: "{{ './' + ansible_distribution + '_' + ansible_distribution_major_version + '/recommendations.yml' }}"
```

Comment:
I think @pcahyna already mentioned this, but the jinja2 string concatenation operator is `~` not `+` - see https://jinja.palletsprojects.com/en/2.10.x/templates/#math - not sure what you guys decided to do about this.  Since `+` appears to be working, no need to change it immediately.